tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -130,6 +130,25 @@ def configure_module_item_caller(context, pyramid_request):
     return views.configure_module_item()
 
 
+def vitalsource_lti_launch_caller(context, pyramid_request):
+    """
+    Call BasicLTILaunchViews.vitalsource_lti_launch().
+
+    Set up the appropriate conditions and then call
+    BasicLTILaunchViews.vitalsource_lti_launch(), and return whatever
+    BasicLTILaunchViews.vitalsource_lti_launch() returns.
+    """
```

Comment:
This comment follows the pattern of the similar functions above, although it feels highly redundant as it just restates the code.